File: smarthomelibary/shlibrary/library/src/main/java/ru/smarthome/library/ArduinoDevice.java
package ru.smarthome.library;

import java.util.List;

public class ArduinoDevice {

    public String name;
    public String description;
    public long guid;
    public List<BaseController> controllers;

    @Override
    public String toString() {
        return "name=" + name +", controllers=" + controllers;
    }
}

File: raspberry/arduinodevices/src/main/java/smarthome/raspberry/arduinodevices/domain/ArduinoDevice.kt
package smarthome.raspberry.arduinodevices.domain

import smarthome.library.common.BaseController
import smarthome.library.common.Id
import smarthome.library.common.IotDevice

open class ArduinoDevice(
    id: Id,
    name: String,
    description: String?,
    controllers: List<BaseController>,
    val ip: String
) : IotDevice(id = id, name = name, description = description, controllers = controllers)